### PR TITLE
API Contracts for Contributions

### PR DESCRIPTION
**Pull request by sumitd94**

### CODE REVIEW

File: contributions/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+```
+{
+  prList: [{
+    'prTitle': <title of pull request>,
+    'prUrl': <url of pull request>,
+    'state': <open or closed>,
+    'prCreatedAt': <unix_timestamp>,
+    'prUpdatedAt': <unix_timestamp>,
+    'prRaisedBy': <username of the assignee>
```
Comment (ankushdharkar) [inline contributions/README.md:13]: Do we need to have a "pr" prefix in the keys? The 'pr' prefix feels redundant. We know it's a PR object, so having simple `title`, `URL`, `createdAt`, etc should suffice

File: contributions/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  }],
+  taskDetails: {
```
Comment (ankushdharkar) [inline contributions/README.md:15]: Can we shrink this down to `task`?

File: contributions/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    'participants': [
+      {
+        'first_name': <first name of user>,
+        'last_name': <last name of user>,
+        'img': <image url of user>
+      },
+      {
+        'first_name': <first name of user>,
+        'last_name': <last name of user>,
+        'img': <image url of user>
```
Comment (ankushdharkar) [inline contributions/README.md:30]: No `username`? Seems important

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: contributions/README.md (hunk 1 above)
Comment (sumitd94) [reply contributions/README.md]: Ok Makes sense, I have updated the object accordingly 9ad7b536cc2fa4ca074c4a616844d1c2e5389ffc

File: contributions/README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+# Users
```
Comment (swarajpure) [inline contributions/README.md:1]: What is this guy doing here? 😅

File: contributions/README.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  }],
+  task: {
+    'title': <task tile>,
+    'purpose': <why this task is needed>,
+    'endsOn': <unix_timestamp>,
+    'startedOn': <unix_timestamp>,
+    'deployedOn': <unix_timestamp>,
+    'participants': [
+      {
+        'first_name': <first name of user>,
```
Comment (swarajpure) [inline contributions/README.md:23]: Mix of snake_case and camelCase...
Can we stick to one convention?

File: contributions/README.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+
+```
+{
+  message: 'Contributions returned successfully!'
```
Comment (swarajpure) [inline contributions/README.md:63]: Trailing comma missing here 😬

**Review by swarajpure — CHANGES_REQUESTED** [review]
More APIs 😋

Requesting some minor changes. Thank you! 😁

File: contributions/README.md (hunk 5 above)
Comment (sumitd94) [reply contributions/README.md:23]: Sure !

File: contributions/README.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+      <contribution_object>,
+      <contribution_object>
+    ]
+  }
+}
+```
+
+- **Error Response:**
+  - **Code:** 404
+    - **Content:** `{ error: 'Not Found', message: 'User doesn't exist' }`
```
Comment (Rucha1499) [inline contributions/README.md:79]: Can we have the word `error` and `message` in single quotes here, just to make it look similar to the one written below? 
Thank you!

**Review by Rucha1499 — COMMENTED** [review]
Great work! @sumitd94 🎉 

Requesting very minor changes! Thank you. 😃

File: contributions/README.md (hunk 7 above)
Comment (ankushdharkar) [reply contributions/README.md:79]: @Rucha1499 JSON **requires** double quotes for keys and values.

It's only when we create an object in JavaScript, that we can get away with quotes for keys, and single quotes for values

Comment (Rucha1499) [reply contributions/README.md:79]: Okay. Got it! 👍

Comment (sumitd94) [reply contributions/README.md]: Fixed the changes.

**Review by ankurnarkhede — APPROVED** [review]

**Review by swarajpure — APPROVED** [review]

**Pull request merged**